core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -789,7 +789,7 @@ private ExecutionModel parseCreateMatView(
         String baseTableName = null;
         if (isWithKeyword(tok)) {
             expectTok(lexer, "base");
-            baseTableName = Chars.toString(tok(lexer, "base table expected"));
+            baseTableName = Chars.toString(unquote(tok(lexer, "base table expected")));
```

Comment:
you should probably also remove a `public.` prefix if it's present. since we have to treat a table `public.foo` as if it was just `foo`.

Reply:
i was not aware of having a `public` schema by default.
are we doing this already for table names?

Reply:
yeah, we are actually handling the public schema.
good catch.
let me add it.